buzzmobile/drive/car_interface/src/main.cpp

```diff
@@ -34,13 +34,13 @@ int main(int argc, char **argv) {
 
   //arduino.open("/dev/arduino_motor_controller", 9600);
   arduino.open("/dev/ttyACM0", 9600);
-  //arduino.setOdometryCallback(odometry_callback);
+  arduino.setOdometryCallback(odometry_callback);
 
   ros::NodeHandle node_handle;
 
   last_command_time = ros::Time::now();
 
-  //encoder_pub = node_handle.advertise<core_msgs::Odom>("encoder_odom", 1000);
+  encoder_pub = node_handle.advertise<buzzmobile::CarPose>("measured_pose", 1000);
```

Comment:
Can you rename to `odom_car_pose` or `measured_car_pose`?

Also do we want a queue of 1000?